sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -557,7 +557,13 @@ class AstBuilder extends DataTypeAstBuilder with SQLConfHelper with Logging {
       notMatchedBySourceActions.toSeq)
   }
 
-  /**
+  override def visitMultipartIdentifierList(
+      ctx: MultipartIdentifierListContext): Seq[UnresolvedAttribute] = withOrigin(ctx) {
```

Comment:
Since we have dedicated code to resolve variable in EXECUTE IMMEDIATE, I think we can use `Seq[String]` directly instead of `UnresolvedAttribute`

Reply:
in USING we can either have literal or variable, and we need to distinguish between these two in SubstituteExecuteImmediate rule, hence i think its cleaner to have AST node than to pass Seq[String] directly and also I wouldn't be able to apply pattern matching